tests/unit/test-util/test.updates.js

```diff
@@ -0,0 +1,28 @@
+/* @flow */
+import {it, describe} from 'mocha';
+import {assert} from 'chai';
+import sinon from 'sinon';
+
+import {checkForUpdates} from '../../../src/util/updates';
+
+describe('util/updates', () => {
+  describe('checkForUpdates()', () => {
+    it('calls the notifier with the correct parameters', () => {
+      const updateNotifierStub = sinon.spy(() => {
+        return {
+          notify: sinon.spy(),
+        };
+      });
+
+      checkForUpdates({
+        version: '1.0.0',
+        updateNotifier: updateNotifierStub,
+      });
+      assert.equal(updateNotifierStub.called, true);
+      assert.equal(updateNotifierStub.firstCall.args[0].pkg.name, 'web-ext');
+      assert.equal(updateNotifierStub.firstCall.args[0].pkg.version, '1.0.0');
+      assert.equal(updateNotifierStub.firstCall.args[0].updateCheckInterval,
+                  1000 * 60 * 60 * 24 * 7);
```

Comment:
Sorry if I wasn't clear about this style rule. We always align the second argument to the inside of the open parenthesis which allows the reader to easily see that the statement within parentheses continues on the next line:
````javascript
      assert.equal(updateNotifierStub.firstCall.args[0].updateCheckInterval,
                   1000 * 60 * 60 * 24 * 7);
````

We don't have the lint rule for this enabled because it causes some other problems.